src/dotnet/commands/dotnet-help/xlf/LocalizableStrings.de.xlf

```diff
@@ -199,12 +199,12 @@
       </trans-unit>
       <trans-unit id="SDKListSdksCommandDefinition">
         <source>Display the installed SDKs.</source>
-        <target state="new">Display the installed SDKs.</target>
+        <target state="translated">Die installierten SDKs anzeigen.</target>
         <note />
       </trans-unit>
       <trans-unit id="SDKListRuntimesCommandDefinition">
         <source>Display the installed runtimes.</source>
-        <target state="new">Display the installed runtimes.</target>
+        <target state="translated">Die installierten Runtimes anzeigen.</target>
```

Comment:
Interestingly, "runtimes" is used as loanword here (> "Runtimes") wheres it is translated to the literal "run time" ("Laufzeit") in other strings, making it inconsistent with the other tralsations.
While I do like the loanword better than the literal translation, it would be better to be consistent. AFAIK the classic .NET doc translations use "Laufzeit" or "Laufzeitumgebung" (=> "run time environment").

Reply:
Hi, there are 3 terminology concepts for this term "runtime".
Translation in this case is correct.

1. „Use DE "Runtime" when referring to the Common Language Runtime (CLR).
2. If not referring to the CLR but to the time of execution, "Laufzeit" is fine (that's a different concept though).”
Plus a third concept, where runtime is not explicitly referring to the CLR, but still refers to an engine. This is what we are looking at here. For this concept, we suggests to use "Runtime" as well, so this particular instance can remain as it is.

Reply:
update commited

Reply:
@janbaltus great, thanks for sharing.
When I wrote "consistent" I was referring to already translated strings still containing "Laufzeit" (literal "run time") - https://github.com/dotnet/cli/search?utf8=✓&q=laufzeit&type=

Reply:
We used Runtime as we think the context is this:
 1.„Use DE "Runtime" when referring to the Common Language Runtime (CLR).
Does the source string refer to Common Language Runtime (CLR)?

Reply:
Yes "Runtime" here is absolutely correct with the rules you menentioned, i just wanted to note that other strings that are not part of this PR will need re-translation using these new rules. (which btw I really like!)

Reply:
Translation "Runtimes" is used when referring to 
The engine at the core of managed code execution. The runtime supplies managed code with services such as cross-language integration, code access security, object lifetime management, and debugging and profiling support.

Translation "Laufzeit" is used when referring to 
The time during which an application is running. During run time, you can interact with an application.

we assume that  other occurences of Laufzeit refer to "The time during which an application is running. During run time, you can interact with an application."

Please correct me if concept is different.

Reply:
I think these strings need to be changed as well:
https://github.com/dotnet/cli/blob/5c35438cfe1ac17fd4c748b8be6c2818d1238807/src/dotnet/commands/dotnet-publish/xlf/LocalizableStrings.de.xlf#L40-L42
https://github.com/dotnet/cli/blob/5c35438cfe1ac17fd4c748b8be6c2818d1238807/src/Microsoft.DotNet.Cli.Utils/xlf/LocalizableStrings.de.xlf#L217-L219
https://github.com/dotnet/cli/blob/5c35438cfe1ac17fd4c748b8be6c2818d1238807/src/dotnet/commands/dotnet-help/xlf/LocalizableStrings.de.xlf#L80-L82
https://github.com/dotnet/cli/blob/e8f7ddd01b63bd0e2b518b974a5aa2d00cd39d60/src/dotnet/xlf/CommonLocalizableStrings.de.xlf#L647-L649

Reply:
PR update with the fix for DEU Runtime